test/e2e/case23_invalid_field_test.go

```diff
@@ -73,7 +73,7 @@ var _ = Describe("Test an objectDefinition with an invalid field", Ordered, func
 			}
 
 			return startTime.After(compPlcEvents[len(compPlcEvents)-1].LastTimestamp.Time)
-		}, defaultTimeoutSeconds, 1).Should(BeTrue())
+		}, 30, 1).Should(BeTrue())
```

Comment:
@yiraeChristineKim did you forget to update this value and the other value in this file to use the new global variable?

Reply:
I think I missed these 2 Thanks Let me open pr